src/screens/Authentication/Notification.js

```diff
@@ -59,12 +59,7 @@ const Notification = () => {
         createdAt: firestore.Timestamp.fromDate(new Date()),
       };
       if (provider && providerCredential) {
-        const userCredential = await auth()
-          .signInWithCredential(providerCredential)
-          .catch(console.error);
-        if (!userCredential) {
-          return setError('An unknown error occurred. Try again');
-        }
+        await auth().signInWithCredential(providerCredential).catch(console.error);
```

Comment:
The response from this is typed to;

`Promise<UserCredential {
    /**
     * Any additional user information assigned to the user.
     */
    additionalUserInfo?: AdditionalUserInfo;
    /**
     * Returns the {@link auth.User} interface of this credential.
     */
    user: User;
  }>`

and throws so not sure sure that not checking it is correct